src/net/socket_utils.zig

```diff
@@ -7,7 +7,7 @@ const Channel = @import("../sync/channel.zig").Channel;
 const std = @import("std");
 const Logger = @import("../trace/log.zig").Logger;
 
-pub const SOCKET_TIMEOUT_US: usize = 1 * std.time.us_per_s;
+pub const SOCKET_TIMEOUT_MS: usize = 1 * std.time.ms_per_s;
```

Comment:
Why the change to milliseconds? All the usage sites still interpret this value as being in microseconds.